config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -77,11 +77,6 @@ extensions:
       script: |-
         sudo yum install -y atomic-openshift-utils
     - type: "script"
-      title: "check out the correct branch of aos-cd-jobs"
-      repository: "aos-cd-jobs"
-      script: |-
-        git checkout sjb
```

Comment:
it is, but we already checkout the `sjb` branch before -> https://github.com/jhadvig/aos-cd-jobs/blob/cleanup/config/test_cases/test_branch_origin_extended_conformance_install_update.yml#L14